src/main/java/dev/joopie/jambot/repository/album/AlbumRepository.java

```diff
@@ -0,0 +1,14 @@
+package dev.joopie.jambot.repository.album;
+
+import dev.joopie.jambot.models.album.Album;
+
+import javax.xml.bind.ValidationException;
+
+public interface AlbumRepository {
+    default AlbumFinder find() {
+        return new AlbumFinder();
+    }
+
+    Album save(Album album) throws ValidationException;
```

Comment:
Is het standaard om in Ebean de ValidationException voor xml te gebruiken? Anders gewoon custom exception maken.